frontend/src/components/pf-overrides/TruncateNoMinWidth.scss

```diff
@@ -0,0 +1,5 @@
+// remove this file when https://github.com/patternfly/patternfly/issues/6062 is solved
```

Comment:
**TL;DR: The bug link is incorrect and there is no PF issue submitted in the PF repo currently, I can submit one.**

I did some research on this. The linked PF issue is resolving another truncate bug in the `Form` component, but not the `Truncate` component itself, so the link here is incorrect.

Then I found there are some use cases (e.g. [here](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetails.tsx#L131)) in the code saying:
> {/* TODO: Remove the custom className after upgrading to PFv6 */}

Then I traced it down and found this ticket: https://issues.redhat.com/browse/RHOAIENG-8837, in this ticket description, it mentions:

> A fix for this issue is being worked on by the Patternfly team: [Patternfly PR #6737](https://github.com/patternfly/patternfly/pull/6737)

However, after checking this PF PR, I found it's still not solving our problem. The problem here is: When we want to use the `Truncate` component, there is always a 6ch min-width, which will make those short string take some unnecessary spaces (See the middle breadcrumb item below).

<img width="520" alt="Screenshot 2024-06-20 at 3 01 10 PM" src="https://github.com/user-attachments/assets/f61c8dfb-b36d-400b-b922-35d7e23541b4">

I even tested it with the PF v6, looks like the problem is still there.

<img width="1511" alt="Screenshot 2024-09-11 at 3 13 00 PM" src="https://github.com/user-attachments/assets/93ff4730-4621-4a61-8de8-c4d7edc7a92b">

I guess the custom style was originally created due to the use case as mentioned in https://github.com/patternfly/patternfly/issues/6062 but then it's widely used in some other places in our code base (because this solution can probably solve different bugs), and the link in the comment is not updated. I can submit an issue in the PF repo and post it here later.

Hope that helps :)

Reply:
Based on the discussion in [Slack](https://redhat-internal.slack.com/archives/C04JMHKSD9C/p1725900825305459?thread_ts=1725652364.721779&cid=C04JMHKSD9C), do we want to keep this override custom style? Feels like the PF team is not going to change anything about it.

Reply:
Whelp, I guess I'll remove the link... Imma start [a thread in our channel](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1726144741794499) to see if we can determine a collective way to move forward here.

Reply:
Dropping support for this component and going with a pure app css solution using PF variables / classNames.